cmd/shield/main.go

```diff
@@ -1485,11 +1486,23 @@ func main() {
 				return err
 			}
 
-			if err := RunJob(id, string(b)); err != nil {
+			var taskUUID string = ""
+			if taskUUID, err = RunJob(id, string(b)); err != nil {
 				return err
 			}
 
-			OK("Scheduled immediate run of job")
+			if taskUUID != "" && *opts.Raw {
+				RawJSON(map[string]interface{}{
+					"ok":        "Scheduled immediate run of job",
+					"task_uuid": taskUUID,
+				})
+			} else {
+				OK("Scheduled immediate run of job")
+				if taskUUID != "" {
+					ansi.Printf("To view task, type @B{shield show task %s}\n", taskUUID)
+				}
+			}
+
```

Comment:
I think these comments are all out of date, but i'm not sure. OK isn't called when options.Raw is specified here.. so the issue would be on the RawJSON call passing in task_uuid of an empty string variable.. which should be fine, since it's just `""`.